rfc6962-bis.xml

```diff
@@ -1177,6 +1177,9 @@ entry specified by <spanx style="verb">start</spanx>.
           </t>
         </list>
       </t>
+	  <t>
+        Additionally, a TLS server which supports presenting SCTs using an OCSP response MAY provide it when the TLS client included the <spanx style="verb">status_request_v2</spanx> extension (<xref target='RFC6961'/>) in the (extended) <spanx style="verb">ClientHello</spanx>, but only in addition to the <spanx style="verb">status_request</spanx> extension mentioned above.
```

Comment:
It seemed the two were somewhat bound, but I may well have misunderstood. I suppose a TLS server *could* support just one of the two extensions, even though they send the same message back?

For the indentation of this paragraph, it seems okay to me, it was just the surrounding tags that were wrong.